src/map/pc.c

```diff
@@ -10567,6 +10569,11 @@ static int pc_unequipitem(struct map_session_data *sd, int n, int flag)
 		pc->unequipitem(sd, sd->equip_index[EQI_AMMO], PCUNEQUIPITEM_FORCE);
 #endif
 
+	if (sd->inventory_data[n] != NULL) {
+			if (sd->inventory_data[n]->type == IT_AMMO && (sd->inventory_data[n]->nameid != ITEMID_SILVER_BULLET || sd->inventory_data[n]->nameid != ITEMID_SANCTIFIED_BULLET || sd->inventory_data[n]->nameid != ITEMID_SILVER_BULLET_))
```

Comment:
look like code offset bit wrong

Reply:
wait what. O_o
VS seems to be bugging my tabbing setup.